src/main/kotlin/blackjack/domain/player/Participants.kt

```diff
@@ -5,24 +5,59 @@ import blackjack.domain.card.CardsDeck
 class Participants(
     val dealer: Dealer,
     val players: List<Player>,
-    cardsDeck: CardsDeck,
 ) {
 
-    init {
-        val allPlayers = listOf(dealer) + players
+    fun existsBlackjack(): Boolean {
+        return getAllPlayers().any { participant -> participant.isBlackjack() }
+    }
 
-        allPlayers.forEach { player ->
-            repeat(FIRST_CARD_DIVIDE_COUNT) {
-                player.addCard(cardsDeck.divide())
+    fun getReceiveCardStatistics(): List<ReceiveCardStatistics> {
+        return getAllPlayers().map { participant ->
+            val cardNames = participant.cards.map { card ->
+                "${card.denomination.denomination} ${card.pattern}"
             }
+
+            ReceiveCardStatistics(
+                playerName = participant.name,
+                cardNames = cardNames,
+                cardSum = participant.getCardSum(),
+            )
+        }
+    }
+
+    fun getRevenueStatistics(): List<RevenueStatistics> {
+        return getAllPlayers().map { participant ->
+            RevenueStatistics(
+                playerName = participant.name,
+                revenue = participant.getRevenue(),
+            )
         }
     }
 
-    fun getAllPlayers(): List<Participant> {
+    private fun getAllPlayers(): List<Participant> {
```

Comment:
해당 내용을 함수가 아닌 프로퍼티와 같이 선언해도 좋을 것 같아요!

```suggestion
    private val allPlayers: List<Participant> = listOf(dealer.dealer) + players.map { it.player }
```

Reply:
이게 더 좋을것 같군요! 감사합니다